unitstatushandler/softwaremanager.go

```diff
@@ -1239,6 +1244,35 @@ func (manager *softwareManager) removeServices() (removeErr string) {
 }
 
 func (manager *softwareManager) runInstances(newServices []string) (runErr string) {
+	manager.InstanceStatuses = []cloudprotocol.InstanceStatus{}
```

Comment:
Now I see from where empty `runState` came from.  Probably better to send `activating` status for instances and remove empty `runState` functionality.

Reply:
Done